The original packaging script installed relative to itself, not needed here.
--- a/libsearpc.pc.in
+++ b/libsearpc.pc.in
@@ -1,4 +1,4 @@
-prefix=(DESTDIR)@prefix@
+prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
 includedir=@includedir@
--- a/Makefile.am
+++ b/Makefile.am
@@ -18,16 +18,5 @@
 
 SUBDIRS = lib pysearpc ${MAKE_DEMO} tests
 
-install-data-local:
-if MACOS
-	sed -i '' -e "s|(DESTDIR)|${DESTDIR}|g" $(pcfiles)
-else
-if FBSD
-	sed -i '' -e "s|(DESTDIR)|${DESTDIR}|g" $(pcfiles)
-else
-	${SED} -i "s|(DESTDIR)|${DESTDIR}|g" $(pcfiles)
-endif
-endif
-
 dist-hook:
 	git log -1 > $(distdir)/latest_commit
